Vision/proposal.md

```diff
@@ -0,0 +1,62 @@
+# The Mission of the W3C
+
+The W3C leads the community in defining a World Wide Web that is inclusive and respectful of its users, 
+to empower an ethical, equitable, informed, and interconnected society.
```

Comment:
```suggestion
to empower an equitable, informed, and interconnected society.
```
I'm not sure what an "ethical" society is; I think our work definitely should be ethical, but I'm not sure how credible we can be in claiming our work helps make society "ethical".

Reply:
re-filed in https://github.com/w3c/AB-public/pull/111/files#r1302738211